source/Classroom/Troubleshooting/Account_Administration/dont_know_your_username.md

```diff
@@ -1,12 +1,12 @@
 ---
 seo:
   title: Don't know your username?
```

Comment:
Based on our discussion on Friday afternoon, I would call this page something like: Recovering your username